.buildkite/gem-push.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -eu
+
+GEM_NAME="danger-dangermattic"
+
+echo "--- :hammer: Build Gemspec"
+gem build "$GEM_NAME.gemspec" -o "$GEM_NAME.gem"
```

Comment:
The `.gemspec` file is currently still named `dangermattic.gemspec` not `danger-dangermattic.gemspec`. We might want to rename it